examples/model_selection/plot_tuned_decision_threshold.py

```diff
@@ -116,17 +133,7 @@
 from sklearn.model_selection import TunedThresholdClassifierCV
 
 tuned_model = TunedThresholdClassifierCV(estimator=model, scoring="balanced_accuracy")
```

Comment:
OK so what we could do is maximize the F1-score and thus compute the precision/recall/f1-score. The story telling would be that we could maximize the F1-score but that the mean between the precision and recall (the definition of f1-score) is a bit arbritrary and that in practice medical doctors are actually looking at a trade-off between the precision and recall and thus the motivation of maximizing a metric under constraint.